code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -0,0 +1,236 @@
+			shoes = /obj/item/clothing/shoes/sneakers/black
+		if(4)
+			flavour_text += "you were always joked about by your friends for \"not playing with a full deck\", as they so <i>kindly</i> put it. It seems that they were right when you, on a tour \
+			at one of Nanotrasen's state-of-the-art research facilities, were in one of the escape pods alone and saw the red button. It was big and shiny, and it caught your eye. You pressed \
+			it, and after a terrifying and fast ride for days, you landed here. You've had time to wisen up since then, and you think that your old friends wouldn't be laughing now.</b>"
+			uniform = /obj/item/clothing/under/color/grey/glorf
+			shoes = /obj/item/clothing/shoes/sneakers/black
+			back = /obj/item/weapon/storage/backpack
+	..()
+
+//Broken rejuvenation pod: Spawns in animal hospitals in lavaland. Ghosts become disoriented interns and are advised to search for help.
+/obj/effect/mob_spawn/human/doctor/alive/lavaland
+	name = "broken rejuvenation pod"
+	desc = "A small sleeper typically used to instantly restore minor wounds. This one seems broken, and its occupant is comatose."
+	mob_name = "a translocated vet"
+	flavour_text = "<font size=3><b>W</b></font><b>hat...? Where are you? Where are the others? This is still the animal hospital - you should know, you've been an intern here for weeks - but \
+	everyone's gone. One of the cats scratched you just a few minutes ago. That's why you were in the pod - to heal the scratch. The scabs are still fresh; you see them right now. So where is \
+	everyone? Where did they go? What happened to the hospital? And is that <i>smoke</i> you smell? You need to find someone else. Maybe they can tell you what happened.</b>"
+
+
+//Shattered prisoner containment sleeper: Spawns in crashed prison ships in lavaland. Ghosts become escaped prisoners and are advised to find a way out of the mess they've gotten themselves into.
+/obj/effect/mob_spawn/human/prisoner_transport
+	name = "shattered prisoner containment sleeper"
+	desc = "A sleeper designed to put its occupant into a deep coma, unbreakable until the sleeper turns off. This one's glass is shattered and you see a pale, sleeping face staring out."
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	mask = /obj/item/clothing/mask/breath
+	shoes = /obj/item/clothing/shoes/sneakers/orange
+	pocket1 = /obj/item/weapon/tank/internals/emergency_oxygen
+	roundstart = FALSE
+	death = FALSE
+	flavour_text = "<font size=3><b>G</b></font><b>ood. It seems as though your ship crashed. You're a prisoner, sentenced to hard work in one of Nanotrasen's labor camps, but it seems as \
+	though fate has other plans for you. You remember that you were convicted of "
+
+/obj/effect/mob_spawn/human/prisoner_transport/New()
+	var/list/crimes = list("murder, but whatever they might say, you <i>know</i> that it was self-defense", \
+	"rape, but they're lying since you both agreed first", \
```

Comment:
I tried to be realistic. If a server that's 18+ can't handle it, then I can remove it.